src/main/java/seedu/contax/logic/commands/BatchCommand.java

```diff
@@ -0,0 +1,124 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.contax.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_EQUALS;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_SEARCH_TYPE;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.exceptions.CommandException;
+import seedu.contax.logic.parser.AddressBookParser;
+import seedu.contax.logic.parser.ParserUtil;
+import seedu.contax.logic.parser.exceptions.ParseException;
+import seedu.contax.model.Model;
+import seedu.contax.model.person.Person;
+import seedu.contax.model.util.SearchType;
+
+
+
+
+/**
+ * Batch edit or delete a person identified using base on string =/ provided.
+ */
+public class BatchCommand extends Command {
+    public static final String COMMAND_WORD = "batch";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Perform command in batch \n"
+            + "Parameters: "
+            + "COMMAND (must be valid command without index) "
+            + PREFIX_SEARCH_TYPE + "SEARCH_TYPE "
+            + PREFIX_EQUALS + "PATTERN \n"
+            + "Example: " + COMMAND_WORD + " edit "
```

Comment:
Sample command does not look correct.

Reply:
changed

Reply:
This is still wrong it displays `batch edit by/phone /=123` and edit will fail.

Reply:
it shd follow PREFIX_EQUALS correctly, what command u put?

Reply:
Doesn't `edit` fail if you dont specify something to edit?

Reply:
ohhh add the missing field

Reply:
@HanJiyao This is the last issue
```suggestion
            + "Example: " + COMMAND_WORD + " edit p/12345678 "
```